### PR TITLE
Modified README.md to mirror that of OBJ2GLTF.

### PR DESCRIPTION
**Pull request by JoshuaStorm**
May add some quick command line examples. At the very least I figured a neat command line flag chart, similar to that in the OBJ2GLTF readme, would be useful.

### CODE REVIEW

**Comment by JoshuaStorm** [comment]
Added limitations section. Any thoughts on better phrasing than "...and other functionality."

**Comment by lilleyse** [comment]
The phrasing is fine for now.

**Comment by lilleyse** [comment]
The image at the bottom of the page is missing.

File: README.md [hunk 1]
```diff
@@ -66,6 +79,18 @@ To rebuild, run:
 npm run build
 ```
 
+## Limitations
+
+This tool is still in development. We plan on adding additional features like gzip compression, AO baking, and other functionality.
```
Comment (lilleyse) [inline README.md]: Remove the note about gzip compression.

**Pull request merged**